app/main.py

```diff
@@ -1,3 +1,50 @@
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: (float, int)) -> None:
+        self.km = km
+
+    def __add__(self, other: (int, float, "Distance")) -> "Distance":
```

Comment:
You can use `|` operator for type annotations instead, it's prefered over using tuple-like parentheses

Reply:
Done